src/pages/joblist/components/JobItemList.js

```diff
@@ -0,0 +1,100 @@
+import { useState, useEffect } from 'react';
+import styled from 'styled-components';
+import JobItem from './JobItem';
+
+const JobItemList = () => {
+  const [error, setError] = useState(null);
+  const [itemList, setItemList] = useState([]);
+  const [page, setPage] = useState(1);
+
+  useEffect(() => {
+    fetch(`https://picsum.photos/v2/list?page=${page}&limit=${12}`)
```

Comment:
limit은 따로 빼서 적용시키는 게 유지보수에 좋을 것 같아요.
```suggestion
    const limit = 12;
    fetch(`https://picsum.photos/v2/list?page=${page}&limit=${limit}`)
```